Exoft.Gamification.Api.Data.Core/Entities/File.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Data.Core.Entities
+{
+    public class File : Entity
```

Comment:
Please.... do not add redundant entities... I asked you to  focus on your task....NOT TO ADD NEW ENTITIES!!!!!